### ISSUE TITLE
[Accessibility][Breadcrumb]: Focus indicator is not visible on more actions menu's in contrast Aquatic/Desert themes. 

### ISSUE THREAD

**Issue by VivianSi233** [microsoft/fluentui#23745]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
Fluent UI React 8.77.1
Browser: Edge Version 102.0.1245.44 (Official build) (64-bit)
OS: Win11 Pro, Version: 10.0.22000 Build 22000
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://developer.microsoft.com/en-us/fluentui#/controls/web/breadcrumb

## Actual Behavior
Precondition: Set contrast theme to Aquatic/Desert
Go to https://developer.microsoft.com/en-us/fluentui#/controls/web/breadcrumb, then press tab key to navigate, check focus indicator.

Focus indicator is not visible on breadcrumb more actions menu's in contrast Aquatic/Desert themes.
![image](https://user-images.githubusercontent.com/89379019/176077744-e99e8efb-d103-4927-baf8-e51d4a567fa9.png)
![image](https://user-images.githubusercontent.com/89379019/176078961-b9c95ff2-e212-4f8b-a45b-efca8ce3baaa.png)

## Expected Behavior
Focus indicator should be visible on breadcrumb more actions menu's in contrast Aquatic/Desert themes.

### Requested priority

Normal

### Products/sites affected

https://df.security.microsoft.com/emailentity?f=summary&startTime=2022-06-28T01:43:56Z&endTime=2022-06-28T01:43:56Z&id=f458fb48-ab05-476b-610b-08da586499e3&recipient=joshc@contoso599.onmicrosoft.com&mkt=en&tid=e3d69322-d40b-45c2-b17c-7d6b981d7c7b

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by gouttierre** [comment]
@VivianSi233 - Thanks for filing this issue with us. To set expectations the developers will review this issue once capacity allows.

@TristanWatanabe  - Would you be able to confirm if this is a regression, or if this behavior is an issue? It is an internal partner ask and I am not sure if you need to loop Sarah in for the a11y part.

**Comment by TristanWatanabe** [comment]
Can confirm that this is an issue, focus indicator is not visible on the overflowButton in HCM.
